src/Workleap.DomainEventPropagation.Publishing/EventPropagationPublisherOptionsValidator.cs

```diff
@@ -4,22 +4,13 @@ namespace Workleap.DomainEventPropagation;
 
 public sealed class EventPropagationPublisherOptionsValidator : IValidateOptions<EventPropagationPublisherOptions>
```

Comment:
This will be modified in an another PR so all validation happens here and we can remove DataAnnotations from the EventPropagationPublisherOptions class.